app/uk/gov/hmrc/taxhistory/services/helpers/IabdsOps.scala

```diff
@@ -72,7 +72,7 @@ object IabdsOps {
       case iabd if iabd.`type`.isInstanceOf[Allowances] =>
         Allowance(
           amount = iabd.grossAmount.getOrElse {
-            logger.warn("Iabds grossAmount is blank")
+            logger.info("Iabds grossAmount is blank")
```

Comment:
I've asked myself how useful these statements really are if they don't include any information about which user they refer to. But I'm not sure how much personal information we're allowed to print into the logs. If we were asked to debug a user report of odd behaviour, I'm not sure if a generic message like these would be of any help... there would be dozens all exactly the same

Reply:
To be honest, I agree that these logging messages don't really hold any value. It seems to just be a requirement based on what the API returns? Rather than returning 0 the API may be missing the field instead so we check to make sure a value is put in place. This seems like something that would have been added during development to check functionality but just wasn't removed when changes were committed. I will remove these.